targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_UBLOX_EVK_ODIN_W2/PinNames.h

```diff
@@ -80,93 +80,96 @@ typedef enum {
+    P_C16       = PF_7,   // GPIO-7
+    P_C17       = PF_1,   // I2C-SCL
+    P_C18       = PF_0,   // I2C-SDA
 	  // D
-    P_D1    = PB_12,  // RMII-TXD0
-    P_D2    = PB_13,  // RMII-TXD1
-    P_D3    = PB_11,  // RMII-TXEN
-    P_D4    = PA_7,   // RMII-CRSDV
-    P_D5    = PC_4,   // RMII-RXD0
-    P_D6    = PC_5,   // RMII-RXD1
-    P_D8    = PA_1,   // RMII-REFCLK
+    P_D1        = PB_12,  // RMII-TXD0
+    P_D2        = PB_13,  // RMII-TXD1
+    P_D3        = PB_11,  // RMII-TXEN
+    P_D4        = PA_7,   // RMII-CRSDV
+    P_D5        = PC_4,   // RMII-RXD0
+    P_D6        = PC_5,   // RMII-RXD1
+    P_D8        = PA_1,   // RMII-REFCLK
     // TP
-    P_TP5   = PB_4,   // NTRST
-    P_TP7   = PA_13,  // TMS  SWDIO
-    P_TP8   = PA_15,  // TDI
-    P_TP9   = PA_14,  // TCK  SWCLK
-    P_TP10  = PB_3,   // TDO 
-    //P_TP11,         // BOOT0 		
+    P_TP5       = PB_4,   // NTRST
+    P_TP7       = PA_13,  // TMS  SWDIO
+    P_TP8       = PA_15,  // TDI
+    P_TP9       = PA_14,  // TCK  SWCLK
+    P_TP10      = PB_3,   // TDO
+    //P_TP11,         // BOOT0
 
     // Board Pins
     // A0-A5
-    A0      = PF_6,  // AI4
-    A1      = PA_3,  // AI3
-    A2      = PC_2,  // AI12
-    A3      = PF_7,  // LPOCLK, not AI
-    A4      = PG_4,  // not AI
-    A5      = PE_0,  // not AI
+    A0          = PF_6,  // AI4
+    A1          = PA_3,  // AI3
+    A2          = PC_2,  // AI12
+    A3          = PF_7,  // LPOCLK, not AI
+    A4          = PG_4,  // not AI
+    A5          = PE_0,  // not AI
     // D0-D15
-    D0      = PD_9,   // UART3-RX
-    D1      = PD_8,   // UART3-TX
-    D2      = PA_10,  // UART1-RX
-    D3      = PA_11,  // CAN1-RX
-    D4      = PA_12,  // CAN1-TX
-    D5      = PB_8,
-    D6      = PD_11,  // UART3-CTS
-    D7      = PD_12,  // UART3-RTS
-    D8      = PA_9,   // UART1-TX
-    D9      = PE_9,   // SDCard-CS
-    D10     = PE_11,  // SSEL
-    D11     = PE_14,  // MOSI
-    D12     = PE_13,  // MISO
-    D13     = PE_12,  // SCK
-    D14     = PF_0,   // SDA
-    D15     = PF_1,   // SCL
+    D0          = PD_9,   // UART3-RX
+    D1          = PD_8,   // UART3-TX
+    D2          = PA_10,  // UART1-RX
+    D3          = PA_11,  // CAN1-RX
+    D4          = PA_12,  // CAN1-TX
+    D5          = PB_8,
+    D6          = PD_11,  // UART3-CTS
+    D7          = PD_12,  // UART3-RTS
+    D8          = PA_9,   // UART1-TX
+    D9          = PE_9,   // SDCard-CS
+    D10         = PE_11,  // SSEL
+    D11         = PE_14,  // MOSI
+    D12         = PE_13,  // MISO
+    D13         = PE_12,  // SCK
+    D14         = PF_0,   // SDA
+    D15         = PF_1,   // SCL
     // Internal
-    LED1    = PE_0,   // Red / Mode
-    LED2    = PB_6,   // Green / Switch-1
-    LED3    = PB_8,   // Blue
-    LED4    = D10,
-    SW0     = PF_2,   // Switch-0
-    SW1     = PB_6,   // Green / Switch-1
+    LED1        = PE_0,   // Red / Mode
+    LED2        = PB_6,   // Green / Switch-1
```

Comment:
Is there some weird pin overflow here, since LED2= PB_6 and also SW1 = PB_6. I.e. does it mean SW1 button aka BUTTON2 will not work if the LED works?

Reply:
That is a bit strange...it seems to match the Ublox [documentation](https://www.u-blox.com/sites/default/files/ODIN-W2-mbed-Addendum_SIM_%28UBX-16008405%29.pdf) though.

This test program also built and ran as expected:
```c++
#include "mbed.h"

DigitalOut led2(LED2);
DigitalIn  sw1(SW1, PullUp);

int main() {
    while (true) {
        led2 = !sw1;
        printf("%d\r\n", sw1.read());
    }
}
```

Reply:
Yep, I think the LEDs overrun the button.